### PR TITLE
Temporarily ignore content type

### PR DESCRIPTION
**Pull request by ejona86**
Note that this is _only_ for v0.9.x. Master will continue with the current behavior.

### CODE REVIEW

**Comment by carl-mastrangelo** [comment]
Context?

**Comment by ejona86** [comment]
Fixes #1021

**Comment by ejona86** [comment]
And:
> Temporarily disable content type checking due to lacking content-type from C servers.

File: core/src/main/java/io/grpc/internal/Http2ClientStream.java [hunk 1]
```diff
@@ -204,8 +204,10 @@ private Status checkContentType(Metadata headers) {
       return null;
     }
     contentTypeChecked = true;
+    // Temporarily disable content type checking due to lacking content-type from C servers.
```
Comment (nmittler) [inline core/src/main/java/io/grpc/internal/Http2ClientStream.java]: Maybe add a link to the tracking bug?

**Comment by nmittler** [comment]
LGTM

**Pull request merged**